lib/ansible.js

```diff
@@ -24,10 +24,18 @@ AbstractAnsibleCommand.prototype.exec = function(options) {
 
   var deferred = Q.defer();
   var processOptions = {};
-  if (options && options.cwd) {
+  if (!options) options = {};
+
+  if (options.cwd) {
     processOptions.cwd = options.cwd;
   }
 
+  // Make Ansible output unbuffered so the stdout and stderr `data` events 
+  // pickup output up more regularly
+  var unbuffered = (options.buffered) ? '' : 1;
+  processOptions.env = process.env;
```

Comment:
That's what spawn [defaults to](https://github.com/nodejs/node/blob/c22bcd31f082747522dad355f987845cd7b9f5e8/lib/child_process.js#L314) when nothing is supplied.